src/sentry/conf/server.py

```diff
@@ -1823,6 +1823,7 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
     # True if Relay should drop raw session payloads after extracting metrics from them.
     "organizations:release-health-drop-sessions": False,
     # Enable new release UI
+    "organizations:releases-v2-internal": False,
```

Comment:
small nit: can you fix the ordering of the release feature flags?
It should go v2, v2-banner, v2-internal, v2-st

Reply:
🤷